public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple.txt

```diff
@@ -0,0 +1,3 @@
+### Basic Line Drawing
+
+The following code shows examples of using [Draw Line On Window](/api/graphics#draw-line-on-window-1) to draw lines on a window.
```

Comment:
This description should describe what this example is showing ("multi-coloured lines to create wedges of the screen" or something like that).